Data/4_Data.csproj

```diff
@@ -10,6 +10,27 @@
     <EmbeddedResource Remove="Configurations\IdentityMigrations\**" />
     <None Remove="Configurations\IdentityMigrations\**" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="Migrations\20200527095846_addIsPublicFlagToUser.cs" />
```

Comment:
I think these can be removed (this is only relevant if you remove a migration that is actually in develop already)